core/libretro/libretro.cpp

```diff
@@ -416,7 +444,11 @@ void retro_set_environment(retro_environment_t cb)
 #if !defined(TARGET_NO_THREADS)
       {
          "reicast_threaded_rendering",
-         "Threaded rendering (restart); disabled|enabled",
+#ifdef LOW_END
+         "Threaded rendering (restart); enabled|disabled",
+#else
+				 "Threaded rendering (restart); disabled|enabled",
+#endif
```

Comment:
`Threaded rendering` should be the default for all platforms.

Reply:
like above I will fix for all